### ISSUE TITLE
Nav Block: The navigation structure pane is not usable in the sidebar

### ISSUE THREAD

**Issue by apeatling** [WordPress/gutenberg#19691]
**Describe the bug**
My expectation when using the navigation structure pane in the sidebar is that you can click on different navigation links and have them select in the editor. The trouble is that this changes the editor context and the navigation structure view is lost. This is very disorientating, and there is no clear way to get back to the same view:

![2020-01-15 15 36 09](https://user-images.githubusercontent.com/1464705/72480696-13d7c880-37ad-11ea-888d-ceb6aed0305b.gif)

**To reproduce**
1. Insert a navigation block with a number of links
2. Select the navigation block and use the navigation structure pane in the sidebar
3. Click on any navigation link, the view changes and you lose your place

**Expected behavior**
I expect to be able to click through multiple navigation links and have those links be selected in the editor. Changing the view after the first click is quite disorienting when using this in the sidebar. It makes more sense in the modal where the editor view is being blocked, so you expect it to disappear when you make a selection.

My suggestion is that it is not included in the sidebar view.

**Comment by shaunandrews** [comment]
This same behavior is true of the content structure icon in the top bar — it goes away once you interact with it. I've pondered on this more than a few times, and I want it to act like a layers list in Photohop/Sketch/Figma. However, I do wonder if there's bias with my experience using those design tools.

--

That said, late last year I explored the idea of having the content structure tool bar be more universally accessible. The first exploration of that added a _section_ within the Document/Block inspector:

<img width="1711" alt="image" src="https://user-images.githubusercontent.com/191598/72742466-c88c3400-3b77-11ea-81e4-fa7c80162f54.png">

Alternatively, another exploration tried adding (yet) another sidebar on the left:

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/191598/72742568-fffae080-3b77-11ea-90ed-4999f396c57e.png">

I lean towards the first option as being more viable and valuable, but I don't think either is a great solution.

**Comment by karmatosed** [comment]
I think we have 2 things here and it's good to focus and maybe work on both.

1.  The existing navigator might need iterating: to bring in clicking to focus.
2. Iterate on the navigator.

Regarding the second point, there are a few issues to consider:

- Adding movers: #18014 
- Merging document and navigator: #19770 
-  Iterating information: #17806

I think a solution as a step would be to make the links clickable.

**Comment by MichaelArestad** [comment]
In the near future, I think enhancing the navigator as @karmatosed mentioned to have movers, etc is a step forward. Ultimately, I think the navigator in any of its current forms comes with flaws.

One idea that could help mitigate those issues is the idea of a singular persistent block navigator (much like a layers palette). I proposed it [here](https://github.com/WordPress/gutenberg/issues/22113). I think it would resolve this issue among others if we went that route.

Here's a gif from the issue:
![2020-05-06 08 39 23](https://user-images.githubusercontent.com/1123119/81190586-42083080-8f75-11ea-87ff-99c407c3db0e.gif)
